src/image-display.cc

```diff
@@ -120,6 +120,23 @@ static ExifImageOp GetExifOp(Magick::Image &img) {  // NOLINT
     return {};
 }
 
+// An extended version of Magick::readImages that requests a
+// decoding/raster with a transparent background.
+static void readImagesWithTransparentBackground(
+    std::vector<Magick::Image> *sequence, const std::string &image_spec) {
```

Comment:
`image_spec` is probably easier to understand if we call it `filename`

Reply:
(solved)